src/frontends/tensorflow_common/src/op/bias_add.cpp

```diff
@@ -41,7 +52,7 @@ OutputVector translate_bias_add_op(const NodeContext& node) {
         std::vector<int64_t> axes_unsqueeze;
         for (int64_t dim_ind = 0; dim_ind < value_rank; ++dim_ind) {
```

Comment:
in case of complex type, value shape will contain aux dimension by which we should not unsqueeze bias. So `value_rank = value_shape.rank().get_length() - 1;`

Reply:
Hi @rkazants 
Regarding the recent change - i think the correct approach to handle the rank (in float transformed complex tensors) would be by skipping it in the right order. So instead I have reverted your recommended change and added an additional condition in the logic where we select the axis to squeeze. Do take alook

Reply:
However, do correct me if I am wrong, when we say a new dimension is added for supporting complex types does it get added such that:
Say initial dimension (of real or imag tensor) is [3,4]
Then the corresponding tensor would be 
- [2,3,4] (The assumption I have taken) 
- or [3,4,2]? 

Thanks!

Reply:
Hi @rkazants 
Bringing this up again as a reminder.

Thanks!